adr/0015-carrying-over-applications.md

```diff
@@ -0,0 +1,81 @@
+# 15. Carrying over applications
+
+Date: 2020-08-12
+
+## Status
+
+Accepted
+
+## Context
+
+The current recruitment cycle ends on 18th September 2020. At that point there
+will be some candidates who could benefit from their application being carried
+over to the next cycle. Carrying over an application means the candidate can
+apply to courses in the new recruitment cycle without having to fill in the
+whole application form again.
+
+### Carrying over an application makes sense in the following states
+
+#### Before the application reaches the provider
+
+- Withdrawn
+- Unsubmitted
+- Ready to send to provider
+
+#### After the application can’t progress any further
+
+- Conditions met (recruited)
+- Conditions not met
+- Offer withdrawn
+- Offer declined
+- Application cancelled
+
+### Carrying over an application does not make sense in the following states
+
+#### While the application is already under consideration by the provider
+
+- Awaiting provider decision
+
+#### When the application already has an offer in flight
+
+- Offer
+- Meeting conditions
```

Comment:
What is 'Meeting conditions'? Is it a future state (some conditions have been met while others are pending)?